client/trino-cli/src/test/java/io/trino/cli/TestClientOptions.java

```diff
@@ -92,6 +92,22 @@ public void testServerHttpsUri()
         assertEquals(session.getServer().toString(), "https://test/foo");
     }
 
+    @Test
+    public void testServer443Port()
+    {
+        Console console = createConsole("--server=test:443");
+        ClientSession session = console.clientOptions.toClientSession();
+        assertEquals(session.getServer().toString(), "https://test:443");
+    }
+
+    @Test
+    public void testServerHttpsHostPort()
+    {
+        Console console = createConsole("--server=https://test:443");
+        ClientSession session = console.clientOptions.toClientSession();
+        assertEquals(session.getServer().toString(), "https://test:443");
```

Comment:
Could you add another test case, `--server=http://test:443` goes to `http://test:443`?

Reply:
Funny, I just thought of this as I woke up this morning. Will do.

Reply:
Done, please see latest test ``testServerHttpWithPort443``